src/convertToObject.js

```diff
@@ -6,7 +6,29 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const rules = sourceString.split(';');
+
+  rules.forEach((rule) => {
+    if (!rule.trim()) {
+      return;
+    }
+
+    const [property, value] = rule.split(':');
```

Comment:
Using 'split' directly on the rule without checking for multiple colons can lead to incorrect parsing if a property value contains a colon. Although handling multiple colons is not a requirement, it would be more robust to split by the first colon only.